Fix reopened app cta

megcrenshaw opened the pull request:
The CtA to ensure a teacher can access their reopened application wasn't working (see Jira ticket for more details). I removed the experiment, and then tested locally.

<img width="1010" alt="image" src="https://user-images.githubusercontent.com/9142121/171275064-1b9950ff-2b21-4b9a-a358-f049b1cdc76d.png">

## Links

<!--
  Links to relevant external resources; ie, specification documents, Jira tickets, related PRs, Honeybadger errors, etc.
-->

[PLAT-1810](https://codedotorg.atlassian.net/browse/PLAT-1810)

## Testing story

<!--
  Does your change include appropriate tests?
  If so, please describe how the tests included in this PR are sufficient.
  If not, please explain why this change does not need to be tested.
-->

<!-- Other aspects to consider. Delete any sections that are not relevant to your change. -->

## Deployment strategy

## Follow-up work

<!--
  List (ideally with Jira links) any clean-up or technical debt that will be addressed in future work.
-->

## Privacy

<!--
  1.	Does this change involve the collection, use, or sharing of new Personal Data?
  2.	Does this change involve a new or changed use or sharing of existing Personal Data?
-->

## Security

<!-- Link to Jira task(s) where sensitive security issues are discussed privately. -->

## Caching

## PR Checklist:

<!--
  The final step! Before you create your PR, double-check that everything is in order.
  Change [ ] to [X] during creation to check boxes.
-->

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

davidsbailey reviewed (approved):
code change looks good, but can you say a tiny bit more about what happened here? did we just overlook this instance of the experiment when launching it everywhere else?

megcrenshaw commented:
Good question. I looked at the other PR https://github.com/code-dot-org/code-dot-org/pull/45634/files where I deleted the other mentions of the experiment. The only thing I can think of is I did a search for `'teacher-application-saving-reopening'` but not for `TEACHER_APPLICATION_SAVING_REOPENING`? That theory doesn't totally check out because there were places I deleted the TEACHER_APPLICATION_SAVING_REOPENING mentions (without a TODO comment), but perhaps I found those via other TODO comments.
